src/wp-admin/includes/class-wp-filesystem-ftpext.php

```diff
@@ -122,28 +126,29 @@ public function connect() {
 	 */
 	public function get_contents( $file ) {
 		$tempfile = wp_tempnam( $file );
-		$temp     = fopen( $tempfile, 'w+' );
+		$temphandle = fopen( $tempfile, 'w+' );
 
-		if ( ! $temp ) {
+		if ( ! $temphandle ) {
 			unlink( $tempfile );
 			return false;
 		}
 
-		if ( ! ftp_fget( $this->link, $temp, $file, FTP_BINARY ) ) {
-			fclose( $temp );
+		if ( ! ftp_fget( $this->link, $temphandle, $file, FTP_BINARY ) ) {
+			fclose( $temphandle );
 			unlink( $tempfile );
 			return false;
 		}
 
-		fseek( $temp, 0 ); // Skip back to the start of the file being written to
+		fseek( $temphandle, 0 ); // Skip back to the start of the file being written to.
 		$contents = '';
 
-		while ( ! feof( $temp ) ) {
-			$contents .= fread( $temp, 8192 );
+		while ( ! feof( $temphandle ) ) {
+			$contents .= fread( $temphandle, 8 * KB_IN_BYTES );
 		}
 
-		fclose( $temp );
+		fclose( $temphangle );
```

Comment:
Here there is a typo in the var.

Reply:
Thanks, good catch, updated PR to follow. There probably is a sniff for unused variables available, would have to investigate.